soh/soh/Enhancements/randomizer/randomizer_entrance_tracker.cpp

```diff
@@ -669,8 +669,13 @@ void DrawEntranceTracker(bool& open) {
                 UIWidgets::Tooltip("Highlight the previous entrance that Link came from");
                 UIWidgets::PaddedEnhancementCheckbox("Highlight available", "gEntranceTrackerHighlightAvailable", true, false);
                 UIWidgets::Tooltip("Highlight available entrances in the current scene");
-                UIWidgets::PaddedEnhancementCheckbox("Hide undiscovered", "gEntranceTrackerCollapseUndiscovered", true, true);
+                UIWidgets::PaddedEnhancementCheckbox("Hide undiscovered", "gEntranceTrackerCollapseUndiscovered", true, false);
                 UIWidgets::Tooltip("Collapse undiscovered entrances towards the bottom of each group");
+                bool disableHideReverseEntrances = OTRGlobals::Instance->gRandomizer->GetRandoSettingValue(RSK_DECOUPLED_ENTRANCES) == RO_GENERIC_ON;
+                static const char* disableHideReverseEntrancesText = "This option is disabled because \"Decouple Entrances\" is enabled.";
+                UIWidgets::PaddedEnhancementCheckbox("Hide reverse", "gEntranceTrackerHideReverseEntrances", true, true,
+                                              disableHideReverseEntrances, disableHideReverseEntrancesText, UIWidgets::CheckboxGraphics::Cross, false);                
```

Comment:
One last suggestion: (I just found this out), but if the `PaddedEnhancementCheckbox` has a bottom padding, then the tooltip doesn't render. We can just set it to false for now to get the tooltip to render (not a big deal)

Also we want the default value to be true (matching the default value passed into CVarGet below)

```suggestion
                UIWidgets::PaddedEnhancementCheckbox("Hide reverse", "gEntranceTrackerHideReverseEntrances", true, false,
                                              disableHideReverseEntrances, disableHideReverseEntrancesText, UIWidgets::CheckboxGraphics::Cross, true);
```

Reply:
LMAO, how does that even work? Are you OK with the separator line underneath being that much closer, or should we have a spacer in there?

And how did I miss that default setting? *sigh*

Reply:
Can always manually add the `UIWidgets:Spacer(0);` underneath the tooltip for now to add back the spacing.

Reply:
Done.